README.md

```diff
@@ -31,18 +31,18 @@ After building your image, use [push-to-registry](https://github.com/redhat-acti
   </tr>
 
   <tr>
-    <td>tag</td>
+    <td>tags</td>
     <td>No</td>
     <td>
-      Tag to give to the output image.<br>
+      The tags of the image to build. For multiple tags, seperate by a space. For example, <code>v1 v0.1</code>.<br>
```

Comment:
`v1 v0.1` is a poor example, there's no reason you would give one image those two tags. `"v1 latest"` is fine, or something like `"latest ${{ github.sha }}"` would be a cool example.

Reply:
Yeah agreed :+1: